plugins/modules/mysql_replication.py

```diff
@@ -655,6 +675,56 @@ def main():
             module.exit_json(msg="Replica reset", changed=True, queries=executed_queries)
         else:
             module.exit_json(msg="Replica already reset", changed=False, queries=executed_queries)
+    elif mode == 'changereplication':
+        chm = []
+        result = {}
+        if primary_host is not None:
```

Comment:
I don't like this similar repetition with `changeprimary` mode. I can refactor if this make sense.